### PR TITLE
Email Export when Done Button Update

### PR DESCRIPTION
**Pull request by xbryanc**
Button should now be more apparent and hovering it gives a description of why/how it is useful to the user
The email sent when export is finished will also contain domain name to give more context to the user

@dannyroberts

### CODE REVIEW

**Comment by dannyroberts** [comment]
@xbryanc could we get a screenshot or two here?

File: corehq/apps/export/templates/export/partial/export_download_progress.html [hunk 1]
```diff
@@ -56,11 +56,14 @@
                         {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
                         <button type="button"
                                 class="btn btn-default btn-full-width"
+                                title="For exports that may take a while to prepare"
+                                ng-style="{'background-color':'rgb(0, 180, 150)'}"
```
Comment (dannyroberts) [inline corehq/apps/export/templates/export/partial/export_download_progress.html]: We usually try to stick to predefined button colors/appearances; you can pick from these: https://getbootstrap.com/docs/3.3/css/#buttons-options.

@orangejenny do you have a suggestion for button color for the email me when done button that will show up after clicking "Prepare Download" while the download is still loading? I believe at this point the `btn-primary` "Prepare Download" button is disabled but still on-screen.

Comment (orangejenny) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: I'd just make it `btn-default`

Comment (dannyroberts) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: Yeah, sorry that's what it originally was, and there was UAT feedback that the button was not prominent enough. Maybe it'd help to wait for a screenshot?

Comment (orangejenny) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: got it - yeah, screenshot will help

Comment (xbryanc) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: Sorry for the delay, I couldn't get this part of the server up yesterday for some reason.
download exports picture

Comment (orangejenny) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: offlined with Bryan - he's going to have a look at making the "Collecting Data" button switch to `btn-default` while it's disabled and making the email button `btn-primary`, with the email button then switching to `btn-default` when it's clicked

Comment (xbryanc) [reply corehq/apps/export/templates/export/partial/export_download_progress.html]: ^that's what the latest updates change does, I re-explained in the commit statement as well

File: corehq/apps/export/templates/export/partial/export_download_progress.html [hunk 2]
```diff
@@ -55,12 +55,14 @@
                         </button>
                         {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
                         <button type="button"
-                                class="btn btn-default btn-full-width"
+                                title="For exports that may take a while to prepare"
```
Comment (orangejenny) [inline corehq/apps/export/templates/export/partial/export_download_progress.html]: Can you translate this?

**Review by orangejenny — APPROVED** [review]

**Review by dannyroberts — APPROVED** [review]
Very nice, looks good with these changes. Thanks @xbryanc and @orangejenny!

**Pull request merged**